fabtests/multinode/src/core_coll.c

```diff
@@ -193,10 +204,11 @@ static int sum_all_reduce_test_run()
 	uint64_t done_flag;
 	uint64_t result = 0;
 	uint64_t expect_result = 0;
-	uint64_t data = pm_job.my_rank;
+	uint64_t data = pm_job.my_rank + 1;
```

Comment:
Please separate out the +1 change, as that looks like a separate fix, and it's not clear why + 1 is needed.  Or if it is part of the fabtests changes, update the commit message to help me understand the change.

Reply:
It is to make the test slightly more robust by having data starting 1, so that we can make sure that rank 0 is really participating or not to the collective.
I added a comment in the code and changed the commit message to reflect that.